[33mWARNING [0m juju.client.connection:connection.py:858 unexpected facade SSHServer received from the controller
[32mINFO    [0m pytest_operator.plugin:plugin.py:675 Connecting to existing model concierge-microk8s:testing on unspecified cloud
[33mWARNING [0m juju.client.connection:connection.py:858 unexpected facade SSHServer received from the controller
[33mWARNING [0m juju.client.connection:connection.py:858 unexpected facade SSHSession received from the controller
[33mWARNING [0m juju.client.connection:connection.py:858 unexpected facade SSHTunneler received from the controller
[33mWARNING [0m juju.client.connection:connection.py:858 unexpected facade SSHServer received from the controller
[32mINFO    [0m integration.high_availability.conftest:conftest.py:57 Deploying mysql-k8s and scaling to 3 units
[32mINFO    [0m juju.model:model.py:2284 Deploying local:mysql-k8s-0
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [allocating] waiting: installing agent
  mysql-k8s/1 [allocating] waiting: installing agent
  mysql-k8s/2 [allocating] waiting: installing agent
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [executing] maintenance: installing charm software
  mysql-k8s/1 [executing] maintenance: installing charm software
  mysql-k8s/2 [executing] maintenance: installing charm software
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [executing] unknown: 
  mysql-k8s/1 [executing] unknown: 
  mysql-k8s/2 [executing] unknown: 
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [idle] waiting: waiting to get cluster primary from peers
  mysql-k8s/1 [idle] maintenance: Initialising mysqld
  mysql-k8s/2 [idle] waiting: waiting to get cluster primary from peers
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [executing] maintenance: joining the cluster
  mysql-k8s/1 [idle] active: Primary
  mysql-k8s/2 [idle] waiting: waiting to join the cluster
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [executing] active: 
  mysql-k8s/1 [executing] active: Primary
  mysql-k8s/2 [idle] active: 
[32mINFO    [0m integration.high_availability.conftest:conftest.py:60 Deploying mysql-test-app
[32mINFO    [0m juju.model:model.py:2284 Deploying ch:amd64/jammy/mysql-test-app-85
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-test-app/0 [allocating] waiting: installing agent
[32mINFO    [0m integration.high_availability.conftest:conftest.py:63 Relating mysql-k8s with mysql-test-app
[33mWARNING [0m juju.model:model.py:1666 relate is deprecated and will be removed. Use integrate instead.
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [idle] active: 
  mysql-k8s/1 [idle] active: Primary
  mysql-k8s/2 [idle] active: 
  mysql-test-app/0 [idle] waiting: 
[32mINFO    [0m integration.conftest:conftest.py:31 Getting credentials for the MySQL cluster
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s (waiting for exactly 4 units, current : 3)
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [executing] active: 
  mysql-k8s/1 [executing] active: 
  mysql-k8s/2 [executing] active: 
  mysql-k8s/3 [executing] maintenance: installing charm software
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [idle] active: 
  mysql-k8s/1 [idle] active: Primary
  mysql-k8s/2 [idle] active: 
  mysql-k8s/3 [executing] waiting: waiting to get cluster primary from peers
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [idle] active: 
  mysql-k8s/1 [idle] active: Primary
  mysql-k8s/2 [idle] active: 
  mysql-k8s/3 [executing] active: 
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/3 [executing] active: 
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [idle] active: 
  mysql-k8s/2 [idle] active: 
  mysql-k8s/3 [idle] active: 
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s (waiting for exactly 3 units, current : 4)
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s (waiting for exactly 3 units, current : 4)
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [idle] active: 
  mysql-k8s/1 [idle] active: Primary
  mysql-k8s/2 [idle] active:
[32mINFO    [0m pytest_operator.plugin:plugin.py:790 Model status:

Model    Controller          Cloud/Region        Version  SLA          Timestamp
testing  concierge-microk8s  microk8s/localhost  3.6.9    unsupported  02:13:49Z

App             Version                  Status  Scale  Charm           Channel      Rev  Address         Exposed  Message
mysql-k8s       8.0.42-0ubuntu0.22.04.2  active      3  mysql-k8s                      0  10.152.183.242  no       
mysql-test-app  0.0.2                    active      1  mysql-test-app  latest/edge   85  10.152.183.132  no       Last written value=836

Unit               Workload  Agent  Address     Ports  Message
mysql-k8s/0        active    idle   10.1.72.77         
mysql-k8s/1*       active    idle   10.1.72.75         Primary
mysql-k8s/2        active    idle   10.1.72.76         
mysql-test-app/0*  active    idle   10.1.72.78         Last written value=836

[32mINFO    [0m pytest_operator.plugin:plugin.py:796 Juju error logs:

unit-mysql-k8s-0: 02:06:23 ERROR unit.mysql-k8s/0.juju-log database-peers:2: Cannot connect to the pebble in the mysql container to check binlogs collector
unit-mysql-k8s-0: 02:06:23 ERROR unit.mysql-k8s/0.juju-log database-peers:2: Failed to reconcile binlogs collection during peer relation event
unit-mysql-k8s-1: 02:06:24 ERROR unit.mysql-k8s/1.juju-log database-peers:2: Cannot connect to the pebble in the mysql container to check binlogs collector
unit-mysql-k8s-1: 02:06:24 ERROR unit.mysql-k8s/1.juju-log database-peers:2: Failed to reconcile binlogs collection during peer relation event
unit-mysql-k8s-2: 02:06:25 ERROR unit.mysql-k8s/2.juju-log database-peers:2: Cannot connect to the pebble in the mysql container to check binlogs collector
unit-mysql-k8s-2: 02:06:25 ERROR unit.mysql-k8s/2.juju-log database-peers:2: Failed to reconcile binlogs collection during peer relation event
unit-mysql-k8s-0: 02:06:26 ERROR unit.mysql-k8s/0.juju-log database-peers:2: Cannot connect to the pebble in the mysql container to check binlogs collector
unit-mysql-k8s-0: 02:06:26 ERROR unit.mysql-k8s/0.juju-log database-peers:2: Failed to reconcile binlogs collection during peer relation event
unit-mysql-k8s-0: 02:06:30 ERROR unit.mysql-k8s/0.juju-log database-peers:2: Cannot connect to the pebble in the mysql container to check binlogs collector
unit-mysql-k8s-0: 02:06:30 ERROR unit.mysql-k8s/0.juju-log database-peers:2: Failed to reconcile binlogs collection during peer relation event
unit-mysql-k8s-1: 02:06:33 ERROR unit.mysql-k8s/1.juju-log database-peers:2: Cannot connect to the pebble in the mysql container to check binlogs collector
unit-mysql-k8s-1: 02:06:33 ERROR unit.mysql-k8s/1.juju-log database-peers:2: Failed to reconcile binlogs collection during peer relation event
unit-mysql-k8s-0: 02:06:33 ERROR unit.mysql-k8s/0.juju-log database-peers:2: Cannot connect to the pebble in the mysql container to check binlogs collector
unit-mysql-k8s-0: 02:06:33 ERROR unit.mysql-k8s/0.juju-log database-peers:2: Failed to reconcile binlogs collection during peer relation event
unit-mysql-k8s-2: 02:06:35 ERROR unit.mysql-k8s/2.juju-log database-peers:2: Cannot connect to the pebble in the mysql container to check binlogs collector
unit-mysql-k8s-2: 02:06:35 ERROR unit.mysql-k8s/2.juju-log database-peers:2: Failed to reconcile binlogs collection during peer relation event
unit-mysql-k8s-1: 02:06:38 ERROR unit.mysql-k8s/1.juju-log database-peers:2: Cannot connect to the pebble in the mysql container to check binlogs collector
unit-mysql-k8s-1: 02:06:38 ERROR unit.mysql-k8s/1.juju-log database-peers:2: Failed to reconcile binlogs collection during peer relation event
unit-mysql-k8s-2: 02:06:41 ERROR unit.mysql-k8s/2.juju-log database-peers:2: Cannot connect to the pebble in the mysql container to check binlogs collector
unit-mysql-k8s-2: 02:06:41 ERROR unit.mysql-k8s/2.juju-log database-peers:2: Failed to reconcile binlogs collection during peer relation event

[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Forgetting main...